core/dbt/main.py

```diff
@@ -1084,6 +1084,17 @@ def parse_args(args, cls=DBTArgumentParser):
         '''
     )
 
+    p.add_argument(
+        '-q',
```

Comment:
I'm fine with this, but I want to highlight the short argument for other reviewers to make sure we want that in addition to the long `--quiet`.

Reply:
I like short arguments so I'm happy with this!